### PR TITLE
changes from previous merge comments

### PR DESCRIPTION
**Pull request by Jake-TJX**

### CODE REVIEW

**Review by mattiaconis-tjx — APPROVED** [review]
looks good now

**Review by ievafil — APPROVED** [review]

File: BackEndDev/routes/customers.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var express = require('express');
 var router = express.Router();
 var customers = require('../mockData.json/customers.json');
 const { restart } = require('nodemon');
-const { NotFound } = require('http-errors');
+
```
Comment (mike-Y-TJX) [inline BackEndDev/routes/customers.js:5]: remove this restart require

**Review by mike-Y-TJX — CHANGES_REQUESTED** [review]

File: BackEndDev/routes/customers.js (hunk 1 above)
Comment (Jake-TJX) [reply BackEndDev/routes/customers.js:5]: done

**Review by mattiaconis-tjx — APPROVED** [review]

**Review by mike-Y-TJX — APPROVED** [review]
looks good

**Pull request merged**